src/SignalR/clients/cpp/src/signalrclient/http_client.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include <string>
+#include <functional>
+#include <map>
+#include <chrono>
+
+namespace signalr
+{
+    enum http_method
```

Comment:
```suggestion
    enum class http_method
```

We use C++11 (or higher) right? Seems like we should use a scoped enum if we're building new things. I'd prefer `http_method::GET` over putting `GET` in the global scope.